src/features/mentors/edit-mentor/edit-mentor.js

```diff
@@ -26,6 +36,15 @@ export const EditMentor = ({ id }) => {
     error: mentorError,
   } = useSelector(mentorIdSelector, shallowEqual);
 
+  const [dispatchLoadMentors] = useActions([
+    fetchMentorById,
+    fetchActiveMentors,
+  ]);
```

Comment:
you should better leave just one action creator fetchMentorById, second one is needless.
 ```const dispatchMentorById = useActions(fetchMentorById);```